client/extensions/woocommerce/app/dashboard/index.js

```diff
@@ -168,10 +169,12 @@ function mapStateToProps( state ) {
 	const selectedSite = getSelectedSiteWithFallback( state );
 	const loading =
 		areOrdersLoading( state ) || areSetupChoicesLoading( state ) || areProductsLoading( state );
-	const hasOrders = getOrders( state ).length > 0;
+	const newOrders = getOrders( state );
 	const hasProducts = getTotalProducts( state ) > 0;
 	const productsLoaded = areProductsLoaded( state );
 	const finishedInitialSetup = getFinishedInitialSetup( state );
+	const hasOrders = find( newOrders, { status: 'processing' } );
```

Comment:
You might want to use `isOrderWaitingFulfillment` from `woocommerce/lib/order-status` – I _think_ you can swap in `find( newOrders, isOrderWaitingFulfillment );`